apollo-router/src/plugins/cache/metrics.rs

```diff
@@ -201,7 +201,7 @@ impl CacheCounter {
 
     fn make_filter() -> Bloom<CacheKey> {
         // the filter is around 4kB in size (can be calculated with `Bloom::compute_bitmap_size`)
-        Bloom::new_for_fp_rate(10000, 0.2)
+        Bloom::new_for_fp_rate(10000, 0.2).expect("no OS source of randomness, that's a bit much")
```

Comment:
Worth noting that bloomfilter 1.x did an `.unwrap()` _inside_ the constructor. So this is not a regression.